### PR TITLE
Store ECS Agent state on the host

### PR DESCRIPTION
**Pull request by Tomdarkness**
Prevents the ECS Agent registering as a brand new container instance every time the agent or host is rebooted.

### CODE REVIEW

File: running-coreos/cloud-providers/ecs/index.md [hunk 1]
```diff
@@ -39,10 +39,11 @@ coreos:
        [Service]
        Environment=ECS_CLUSTER=your_cluster_name
        Environment=ECS_LOGLEVEL=warn
+       Environment=ECS_CHECKPOINT=true
        ExecStartPre=-/usr/bin/docker kill ecs-agent
        ExecStartPre=-/usr/bin/docker rm ecs-agent
        ExecStartPre=/usr/bin/docker pull amazon/amazon-ecs-agent
-       ExecStart=/usr/bin/docker run --name ecs-agent --env=ECS_CLUSTER=${ECS_CLUSTER} --env=ECS_LOGLEVEL=${ECS_LOGLEVEL} --publish=127.0.0.1:51678:51678 --volume=/var/run/docker.sock:/var/run/docker.sock amazon/amazon-ecs-agent
+       ExecStart=/usr/bin/docker run --name ecs-agent --env=ECS_CLUSTER=${ECS_CLUSTER} --env=ECS_LOGLEVEL=${ECS_LOGLEVEL} --env=ECS_CHECKPOINT=${ECS_CHECKPOINT} --publish=127.0.0.1:51678:51678 --volume=/var/run/docker.sock:/var/run/docker.sock -v /home/core/ecs-data:/data amazon/amazon-ecs-agent
```
Comment (crawford) [inline running-coreos/cloud-providers/ecs/index.md]: Please use the long flag "--volume=" instead of "-v".

Comment (crawford) [inline running-coreos/cloud-providers/ecs/index.md]: Please bind /var/lib/aws/ecs to /data instead of /home/core/ecs-data.

**Comment by Tomdarkness** [comment]
@crawford Thanks, I'll make the changes. 

Was unsure if there was a CoreOS recommended way of storing data on the host, asked on IRC but did not get a reply. My original choice of directory was from googling and finding this gist: https://gist.github.com/Luzifer/c184b6b04d83e6d6fbe1

**Comment by crawford** [comment]
LGTM. Thanks!

**Pull request merged**